style.css

```diff
@@ -296,116 +344,274 @@ a:active {
   justify-content: space-evenly;
 }
 
-.works-details li {
-  max-width: 100%;
-  width: max-content;
-  background: #fff;
-  border-radius: 10px;
-  padding: 12px;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 12px;
-  line-height: 16px;
-  color: #36b37e;
+footer {
+  background: url("images/footer-desk-right.png") no-repeat right bottom,;
+  margin: auto;
+  gap: 58px;
+  width: 100%;
 }
 
-form {
-  background-color: rgb(255, 255, 255) solid;
-  max-width: 327px;
-  padding: 0.1rem 0 0 0;
-  margin: 0 auto;
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you stick to the design as much as possible.  (font, spacing(`margin, padding`), color ...
- [ ] Kindly you may consider centering the last section of the page. (the form and the side literature.) `hint: You may consider using the flexbox or the grid container to do so.`
- [ ] Also make sure there's no scroll bar at the bottom of the page. the page content has to fit the hundred percent width of the screen size
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201485762-cf31d914-068a-4306-b903-d2aa1064a327.png)

![image](https://user-images.githubusercontent.com/97100091/201485783-ee885235-bcfe-4e1c-b81a-bfe84dde5e50.png)


The Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201485728-a58e239e-e357-41f8-8a25-362ec74a4b28.png)